src/components/WithinBannerContext/context.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Would you mind moving this to `src/utilities/within-banner-context.tsx` we're currently moving some of our shared utilities/contexts over 😄